.github/workflows/continuous-integration-snifftest.yml

```
@@ -0,0 +1,48 @@
+name: basic-test
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_call:
```

```suggestion
  workflow_call:
```
(since `.github/workflows/continuous-integration-stager.yml` will already trigger it ?)